app/test_main.py

```diff
@@ -1,3 +1,16 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize("cat_age, dog_age, expected", [
+    (0, 0, [0, 0]),
+    (14, 14, [0, 0]),
+    (15, 15, [1, 1]),
+    (23, 23, [1, 1]),
+    (24, 24, [2, 2]),
+    (27, 27, [2, 2]),
+    (28, 28, [3, 2]),
+    (100, 100, [21, 17]),
+])
+def test_get_human_age(cat_age: int, dog_age: int, expected: int) -> None:
+    assert get_human_age(cat_age, dog_age) == expected
```

Comment:
Add a separate test to test `get_human_age` in case it receives unexpected arguments with data types other than integers. You should make sure that it raises the correct exception according to the [checklist](https://github.com/Artem-user-19/py-cat-and-dog-years/blob/develop/checklist.md#clean-code).